db/structure.sql

```diff
@@ -44,7 +44,7 @@ CREATE TABLE `bmap_flowcell` (
   KEY `fk_bmap_flowcell_to_study` (`id_study_tmp`),
   CONSTRAINT `fk_bmap_flowcell_to_sample` FOREIGN KEY (`id_sample_tmp`) REFERENCES `sample` (`id_sample_tmp`) ON DELETE RESTRICT ON UPDATE RESTRICT,
   CONSTRAINT `fk_bmap_flowcell_to_study` FOREIGN KEY (`id_study_tmp`) REFERENCES `study` (`id_study_tmp`) ON DELETE RESTRICT ON UPDATE RESTRICT
-) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+) ENGINE=InnoDB AUTO_INCREMENT=19 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
```

Comment:
Slightly nervous about what all these AUTO_INCREMENTS are doing that it's added. Best check what that does?

Reply:
I noticed those this morning. All I did was run the migration so nothing special done. I will check it.

Reply:
Looks like the reason is because I had to run the migration on test. I have rerun the migration and it has removed the auto increment.